src/main/java/sh/platform/config/reader/Config.java

```diff
@@ -19,6 +19,9 @@
 import static sh.platform.config.reader.PlatformVariables.PLATFORM_TREE_ID;
 import static sh.platform.config.reader.PlatformVariables.PLATFORM_VARIABLES;
```

Comment:
I'm still iffy on config.reader as a package name.  If that's typical in Java OK, but it still feels needlessly verbose and weird to me.

Reply:
we can keep the package name just as `sh.platform.config.`
Just the `sh.platform.config` is my favorite option.
And now, about `sh.platform.configreader` together, no. 
That is not a Java way.
https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html